docker/docker-compose.yml

```diff
@@ -5,30 +5,33 @@ services:
     container_name: "${DOCKER_CONTAINER_API}"
     #restart: always
     depends_on:
-      - mysql
+      - mongodb
     networks:
       - cc
     volumes:
       - "${PROJECT_DIR}:/go/src/app"
-    #environment:
-      #VIRTUAL_HOST: "${DOCKER_CONTAINER_API}"
-  mysql:
-    image: mysql
+    environment:
+      VIRTUAL_HOST: "${DOCKER_CONTAINER_API}"
+  mongodb:
+    image: mongo:3.6.3
     container_name: "${DOCKER_CONTAINER_DB}"
     #restart: always
-    environment:
-      MYSQL_DATABASE: "${DB_NAME}"
-      MYSQL_ROOT_PASSWORD: "${DB_PASSWORD}"
+    volumes:
+      - "./data/db:/data/db"
+    command: mongod --smallfiles --logpath=/dev/null
+    networks:
+      - cc
+  nginx-proxy:
```

Comment:
The `nginx-proxy` is handled in the https://github.com/Cultura-Colectiva-Tech/Mictlan project.

Reply:
I don't understand this comment jejeje

Reply:
The proxy via `nginx-proxy` is handled in the Gateway project ([Mictlán](https://github.com/Cultura-Colectiva-Tech/Mictlan)) along with the `linkerd` Gateway. So there is no need of handle the proxy right here in the project.